docs/bokeh/source/conf.py

```diff
@@ -200,8 +200,6 @@
 }
 
 html_sidebars = {
-    "docs/examples/**": [],
-    "docs/gallery": [],
```

Comment:
I think these changes need to be reverted, we do intentionally avoid sidebars on these pages, order that the Bokeh content can "stand alone" to the fullest degree possible.

Reply:
Thanks @bryevdv, this was indeed a bad change.